docs/src/get-started.md

```diff
@@ -0,0 +1,191 @@
+
+JuliaBox provides a pre-installed Jupyter notebook for Julia. You can take a shot at Turing without installing Julia on your machine in few seconds.
+
+### Install Turing.jl
+
+Turing is an officially registered Julia package, so the following should install a stable version of Turing:
+
+```julia
+Pkg.add("Turing")
+
+# Switch to master branch; recommended
+Pkg.checkout("Turing", "master")
+Pkg.build("Turing")
+
+Pkg.test("Turing")
+```
+
+[**Recommended**] If you want to use the latest version of Turing with some experimental features, you can try the following instead:
+
+```julia
+Pkg.update()
+Pkg.clone("Turing")
+Pkg.build("Turing")
+Pkg.test("Turing")
+```
+
+If all tests pass, you're ready to start using Turing.
+
+## Basics
+
+### Introduction
+
+A probabilistic program is Julia code wrapped in a `@model` macro. It can use arbitrary Julia code, but to ensure correctness of inference it should not have external effects or modify global state. Stack-allocated variables are safe, but mutable heap-allocated objects may lead to subtle bugs when using task copying. To help avoid those we provide a Turing-safe datatype `TArray` that can be used to create mutable arrays in Turing programs.
+
+For probabilistic effects, Turing programs should use the `~` notation:
+
+`x ~ distr` where `x` is a symbol and `distr` is a distribution. If `x` is undefined in the model function, inside the probabilistic program, this puts a random variable named `x`, distributed according to `distr`, in the current scope. `distr` can be a value of any type that implements `rand(distr)`, which samples a value from the distribution `distr`. If `x` is defined, this is used for conditioning in a style similar to Anglican (another PPL). Here `x` should be a value that is observed to have been drawn from the distribution `distr`. The likelihood is computed using `logpdf(distr,y)` and should always be positive to ensure correctness of inference algorithms. The observe statements should be arranged so that every possible run traverses all of them in exactly the same order. This is equivalent to demanding that they are not placed inside stochastic control flow.
+
+Available inference methods include  Importance Sampling (IS), Sequential Monte Carlo (SMC), Particle Gibbs (PG), Hamiltonian Monte Carlo (HMC), Hamiltonian Monte Carlo with Dual Averaging (HMCDA) and The No-U-Turn Sampler (NUTS).
+
+### Simple Gaussian demo
+
+Below is a simple Gaussian demo illustrate the basic usage of Turing.jl
+
+```julia
+# Define a simple Normal model with unknown mean and variance.
+@model gdemo(x) = begin
+  s ~ InverseGamma(2,3)
+  m ~ Normal(0,sqrt(s))
+  x[1] ~ Normal(m, sqrt(s))
+  x[2] ~ Normal(m, sqrt(s))
+  return s, m
+end
+```
+
+Note: for the interests of sanity check, some analytical results on the expectation of samples of this model are E[s] = 49/24 and E[m] = 7/6.
```

Comment:
`for the interests of sanity check` -> `for the sake of sanity checking`
and
`some analytical results on the expectation of samples of this model are E[s] = 49/24 and E[m] = 7/6.` ->
```
the expectation of `s` is 49/24 (2.04166666...) and the expectation of `m` is 7/6 (1.16666666...).
```